### DISCUSSION TITLE
Sort spams based on number of comments

### DISCUSSION THREAD

**Discussion by hunggiadao** [ThioJoe/YT-Spammer-Purge#593] [Ideas]
Please add a sort function when displaying spams after each scan, that way obvious spammers are more likely to be seen than someone who is just very excited. It will also allow creators to exclude someone they believe is genuine if they appear at the bottom of the list.

**Comment by Firecul** [comment]
What criteria would you use to sort, that can be done programmatically?

**Reply by hunggiadao** [reply]
The easiest way is to add a comment counter variable for each spammer. Every time their comment is flagged as spam, increment this variable by 1. In the end when you display all the spam comments and accounts, just use a simple sort function to display them in order of most spam comments to least. This should minimize any chance of false positives for the top spammers. For accounts displayed at the bottom, it is easier to see if they are just excited since the default spam threshold of 4 is kind of low.

**Comment by ThioJoe** [comment]
Yea 2.16 will have this feature. ~~I haven't made a beta release for it yet, but you can download the files from the 2.16 branch I just published and try it now if you want: https://github.com/ThioJoe/YT-Spammer-Purge/tree/2.16.0-Beta~~

Update: The 2.16 beta is now available as a pre-release with this feature.

Example:
<img width="682" alt="image" src="https://user-images.githubusercontent.com/12518330/153029712-11f277bc-f361-485b-b57d-538ba25dce2d.png">

**Reply by lloyddunamis** [reply]
_Aaaaaa_ works as intended & really great so far; quickens _(at least my)_ reviewing process!
